recipes/default.rb

```diff
@@ -19,8 +19,10 @@
 
 influxdb_config node['influxdb']['config_file_path'] do
   config node['influxdb']['config']
+  notifies :restart, 'service[influxdb]', :delayed
 end
 
 service 'influxdb' do
-  action [:enable, :start]
+  supports :restart => true
```

Comment:
I don't know why rubocop is complaining it. Because chef documentation they followed the same syntax in examples https://docs.chef.io/resource_service.html.